### DISCUSSION TITLE
Sequelize Snowflake not returning QueryID

### DISCUSSION THREAD

**Discussion by vishu-guntupalli** [sequelize/sequelize#16195] [Help & Questions]
Hello, we are currently using sequelize to query Snowflake and have a use case where we want to cancel long running queries in the DB after an arbitrary amount of time. To cancel the query we need to have the `queryId` which sequelize does not return out of the box. 

 Is there a way to get Sequelize to return the `queryId` ?

**Comment by WikiRik** [comment]
It does not seem that is currently supported. As maintainers we do not have access to an instance to run integration tests so we are not able to test new features for snowflake. But we can in theory look at PRs that add new functionality. See https://github.com/sequelize/sequelize/discussions/15099 for more information about limitations on support for snowflake
